api/v1/runtimecomponent_types.go

```diff
@@ -144,6 +149,17 @@ type RuntimeComponentSpec struct {
 	SecurityContext *corev1.SecurityContext `json:"securityContext,omitempty"`
 }
 
+// Defines the service account
+type RuntimeComponentServiceAccount struct {
+	// Whether the Service Account token should be mounted into the application pods. Default true
```

Comment:
To be consistent with some of the other default booleans:
`Default true` --> `Defaults to true.`

Reply:
The period `.` at the end is needed